catalog-info.yaml

```diff
@@ -1244,3 +1244,40 @@ spec:
         everyone:
           access_level: BUILD_AND_READ
 
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-beats-elastic-agent-serverless-tests
+  description: "Beats serverless tests from elastic-agent repository"
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/beats-serverless-tests-elastic-agent
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: platform-ingest
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: beats-serverless-tests
+      description: "Beats serverless tests from elastic-agent repository"
+    spec:
+      pipeline_file: ".buildkite/serverless.beats.tests.yml"
+      maximum_timeout_in_minutes: 120
+      provider_settings:
+        trigger_mode: none # don't trigger jobs from github activity
+      repository: elastic/elastic-agent
```

Comment:
What's the reason for using `elastic/elastic-agent`?

Reply:
It happened that the Serverless Beats tests are located in the elastic-agent repo and we need to migrate them to beats workflow

Reply:
It's certainly confusing :/

Your idea is to delete the BK pipeline in `elastic/elastic-agent` after this PR has been merged.

Otherwise, I don't understand we use the elastic-agent repository if this new pipeline will trigger another BK pipeline. 

Can you help me to understand this?

Reply:
The beats pipeline on main/release branches will trigger this new `beats-serverless-tests` pipeline that runs Serverless Beats Tests in the elastic-agent pipeline. I'm not going to delete the `elastic/elastic-agent` pipeline. You might have noticed that there are no GitHub triggers and that it can only be triggered by another pipeline and execute `.buildkite/serverless.beats.tests.yml`. The Serless Beats tests will be moved to `.buildkite/serverless.beats.tests.yml` and be triggered by beats workflow and won't affect the elastic-agent pipeline anymore